example/spec/helpers/Providers.js

```diff
@@ -41,17 +41,28 @@ const buildFtpProvider = async (postfix = '') => {
   return provider;
 };
 
-const buildHttpProvider = async (postfix = '') => {
+const fakeProviderPortMap = {
+  http: process.env.PROVIDER_HTTP_PORT ? Number(process.env.PROVIDER_HTTP_PORT) : 3030,
+  https: process.env.PROVIDER_HTTPS_PORT ? Number(process.env.PROVIDER_HTTPS_PORT) : 4040,
+};
+
+const buildHttpOrHttpsProvider = async (postfix = '', protocol = 'http', systemBucket) => {
```

Comment:
I'm confused here about why defaults are specified for the first 2 params if you have to specify the `systemBucket` param.

Reply:
Fair point, I can switch them around.